core/src/main/java/nl/nn/adapterframework/util/JdbcUtil.java

```diff
@@ -1013,13 +1013,13 @@ public static void executeStatement(Connection connection, String query, int par
 
 	public static synchronized Properties retrieveJdbcPropertiesFromDatabase() {
 		if (jdbcProperties == null) {
-			String jmsRealm = JmsRealmFactory.getInstance()
-					.getFirstDatasourceJmsRealm();
+			String jmsRealm = JmsRealmFactory.getInstance().getFirstDatasourceJmsRealm();
 			if (jmsRealm != null) {
 				jdbcProperties = new Properties();
 				JdbcFacade ibisProp = new JdbcFacade();
-				ibisProp.setJmsRealm(jmsRealm);
-				
+				ibisProp.setJmsRealm(jmsRealm); //Use a realm here so it copies over proxied datasources
```

Comment:
Ik snap dit commentaar niet. Is het niet genoeg om een datasource te hebben?
Volgens mij klopt de logica niet, volgt het niet mijn commentaar bij regel 1017, maar zou het dat wel moeten doen.
Zoiets als 
```
if (jmsRealm != null) {
  ibisProp.setJmsRealm(jmsRealm)
} else {
  ibisProp.setDatasource(appConstants.getProperty("jdbc.datasource.default")
}
```

Reply:
Ja eigenlijk wel, maar dit ding moet helemaal niets met een realm doen toch? ipv daarvan met een datasource oid? ik vond het te veel om er vanalles mee te doen dus heb het maar gelaten.